Assets/MRTK/SDK/Features/UX/Scripts/PulseShader/SurfacePulse.cs

```diff
@@ -215,9 +244,10 @@ public void ResetPulseMaterial()
             ApplyPulseRadiusToMaterial(0);
         }
 
-        /////////////////////////////////////////////////////////////////////////////////////////
-        // Pulse control
-        /////////////////////////////////////////////////////////////////////////////////////////
+        #endregion
+
+        #region Pulse Control
+
         public void PulseOnce()
```

Comment:
Do all of these need to be `public`? Can they be self contained to this script?

Reply:
Not all of these methods need to be public.  Updated in the latest changes